internal/server/recon/recon_name_to_types.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
Could you add a README on how this is generated?

Reply:
done